components/cluster/command/reload.go

```diff
@@ -73,6 +74,7 @@ func newReloadCmd() *cobra.Command {
 	cmd.Flags().StringSliceVarP(&gOpt.Roles, "role", "R", nil, "Only start specified roles")
 	cmd.Flags().StringSliceVarP(&gOpt.Nodes, "node", "N", nil, "Only start specified nodes")
 	cmd.Flags().Int64Var(&gOpt.APITimeout, "transfer-timeout", 300, "Timeout in seconds when transferring PD and TiKV store leaders")
+	cmd.Flags().BoolVarP(&ignoreConfigCheck, "ignore-config-check", "", ignoreConfigCheck, "Ignore the config check result")
```

Comment:
Can we add the `ignoreConfigCheck` field to `gOpt` directly? I think `skip-config-check` more properly.

Reply:
I think `ignore` is better than `skip`, `ignore` means we still check the config, but ignore the result (just put it in the audit log). The `skip` should really do nothing at that step.

Reply:
OK, prefer to put the `ignoreConfigCheck` into `operator.Options`.